test/chunking-form/samples/improved-dynamic-chunks/single-dynamic/_expected/es/generated-dynamic.js

```diff
@@ -0,0 +1,4 @@
+import { value } from './main.js';
+export { value } from './main.js';
+
+console.log('dynamic', value);
```

Comment:
This looks how it should for the simple case: The dynamic chunk is importing its value from `main.js` and we avoided the shared chunk. 👍